heureka/ui/src/components/filters/FilterSelect.jsx

```diff
@@ -54,7 +54,7 @@ const FilterSelect = ({ entityName, isLoading }) => {
     // Debounce search term to avoid unnecessary re-renders
     const debouncedSearchTerm = setTimeout(() => {
       setSearchTerm(entityName, value.target.value)
-    }, 500)
+    }, 2000) //Every 2 seconds to avoid unnecessary API calls
```

Comment:
Do you really want to make the user wait 2 seconds before the search is even started? That will not feel good. If you want to avoid unnecessary API calls then I would remove the `onChange` handler and search only on "return" or search button press.

Reply:
Since currentlz manz requests are sent while searching for a specific CVE I increased this time. But I wouldnät delete it, since CVEs are often long, this allows users to see a filtered list even with just a part of the CVE. What do you think? Should I still remove onChange func?

Reply:
I deactivate it, just handle search on Enter and behind search button for now.